zigmod.yml

```diff
@@ -0,0 +1,5 @@
+id: hnjdgzdi5fgqxz3c9374igqivvttkt387jv853l09xdl66jx
+name: scafetch
+license: MIT
+description: a command-line tool to display information about remote git repositories
+root_dependencies:
```

Comment:
Dependencies need to be added here, which are zfetch and ay-arg, links to both are mentioned in gyro.zzz

Reply:
yes i did't added deps to ask u about dependenices source. [zpm](https://github.com/nektro/zigmod/blob/master/docs/commands/zpm_add.md) or [aquila](https://github.com/nektro/zigmod/blob/master/docs/commands/aq_add.md)?

Reply:
After doing some tests, it seems that adding zigmod support is more compilcated.

1) zfetch package on aquila may be outdated, though this is easy to workaround if we use the git package directly
2) it seems zigmod doesnt like if a dep has no build.zig file, which is the case for ay-arg, this needs investigation

Reply:
yes i tested too. i think it is better to draft this, since the problem exists